R/expr__string.R

```diff
@@ -822,3 +822,83 @@ ExprStr_explode = function() {
 ExprStr_parse_int = function(radix = 2, strict = TRUE) {
   .pr$Expr$str_parse_int(self, radix, strict) |> unwrap("in str$parse_int():")
 }
+
+#' Returns string values in reversed order
+#'
+#' @name ExprStr_reverse
```

Comment:
I think these `@name` tags can be removed.

Reply:
Yes this won't matter anymore once #642 is merged. I can already remove them but don't pay attention to R CMD check failures due to this.